starters/next-wordpress-starter/components/page.jsx

```diff
@@ -4,6 +4,24 @@ import Link from "next/link";
 export default function Page({
   page: { title, date, featuredImage, content },
 }) {
+  let rxSrc, imgSrc, rxPath, imgPath;
```

Comment:
We should abstract this out into a reusable utility of some kind. I think there are some other things in the components folder that need this change applied, and as people build on top of this starter there will likely be even more instances where this code snippet would need to be duplicated.

Reply:
I'm issuing a new PR and closing this one. Added this suggestion to it with this code in a new utilities.jsx file